GameFinder.Example/Program.cs

```diff
@@ -41,11 +41,18 @@ private static void Log(string s)
             try
             {
                 var handler = new THandler();
-                if (!handler.FindAllGames())
+                var res = handler.FindAllGames();
+                if (res.HasErrors)
+                {
+                    Log($"{handlerName} has errors:\n{res.ErrorsToString()}");
+                }
+                
+                if (!res.Value)
```

Comment:
Looking good!  Can definitely get the games as I'd like with this and still print the errors